lib/aws-record.rb

```diff
@@ -44,5 +44,8 @@
 
 module Aws
   module Record
+    GEM_VERSION = File.read(File.expand_path('../../VERSION', __FILE__)).strip
+    ENV['AWS_SDK_1P_LIBRARIES'] = [ENV['AWS_SDK_1P_LIBRARIES'],
```

Comment:
Is there a reason to join like this rather than using `+=`?

Reply:
`+=` doesn't work when the value isn't set yet (because nil and string can't be added).